src/dotnet/commands/dotnet-run3/Run3Command.cs

```diff
@@ -49,6 +52,27 @@ public int Start()
                 .ExitCode;
         }
 
+        private void EnsureProjectIsBuilt()
+        {
+            List<string> buildArgs = new List<string>();
+
+            buildArgs.Add("/nologo");
+            buildArgs.Add("/verbosity:quiet");
+
+            if (!string.IsNullOrWhiteSpace(Configuration))
+            {
+                buildArgs.Add($"/p:Configuration={Configuration}");
+            }
+
+            var buildResult = new MSBuildForwardingApp(buildArgs).Execute();
```

Comment:
We do the same thing for invoking restore3 and pack3:

https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-restore3/Program.cs#L105
https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-pack3/Pack3Command.cs#L81